azurerm/resource_arm_private_endpoint_test.go

```diff
@@ -0,0 +1,271 @@
+	client := testAccProvider.Meta().(*ArmClient).network.PrivateEndpointClient
+	ctx := testAccProvider.Meta().(*ArmClient).StopContext
+
+	for _, rs := range s.RootModule().Resources {
+		if rs.Type != "azurerm_private_endpoint" {
+			continue
+		}
+
+		name := rs.Primary.Attributes["name"]
+		resourceGroup := rs.Primary.Attributes["resource_group_name"]
+
+		if resp, err := client.Get(ctx, resourceGroup, name, ""); err != nil {
+			if !utils.ResponseWasNotFound(resp.Response) {
+				return fmt.Errorf("Bad: Get on PrivateEndpointClient: %+v", err)
+			}
+		}
+
+		return nil
+	}
+
+	return nil
+}
+
+func testAccAzureRMPrivateEndpointTemplate_standardResources(rInt int, location string) string {
+	return fmt.Sprintf(`
+resource "azurerm_resource_group" "test" {
+  name     = "acctestRG-%d"
+  location = "%s"
+}
+
+resource "azurerm_virtual_network" "test" {
+  name                = "acctestvnet-%d"
+  resource_group_name = azurerm_resource_group.test.name
+  location            = azurerm_resource_group.test.location
+  address_space       = ["10.5.0.0/16"]
+}
+
+resource "azurerm_subnet" "test" {
+  name                   = "acctestsnet-%d"
+  resource_group_name    = azurerm_resource_group.test.name
+  virtual_network_name   = azurerm_virtual_network.test.name
+  address_prefix         = "10.5.1.0/24"
+
+  disable_private_link_service_network_policies = true
+  disable_private_endpoint_network_policies     = true
+}
+
+resource "azurerm_public_ip" "test" {
+  name                = "acctestpip-%d"
+  sku                 = "Standard"
+  location            = azurerm_resource_group.test.location
+  resource_group_name = azurerm_resource_group.test.name
+  allocation_method   = "Static"
+}
+
+resource "azurerm_lb" "test" {
+  name                = "acctestlb-%d"
+  sku                 = "Standard"
+  location            = azurerm_resource_group.test.location
+  resource_group_name = azurerm_resource_group.test.name
+  frontend_ip_configuration {
+    name                 = azurerm_public_ip.test.name
+    public_ip_address_id = azurerm_public_ip.test.id
+  }
+}
+
+resource "azurerm_private_link_service" "test" {
+  name                           = "acctestpls-%d"
+  location                       = azurerm_resource_group.test.location
+  resource_group_name            = azurerm_resource_group.test.name
+  nat_ip_configuration {
+    name                         = "primaryIpConfiguration-%d"
+    subnet_id                    = azurerm_subnet.test.id
+  }
+  load_balancer_frontend_ip_configuration_ids = [
+    azurerm_lb.test.frontend_ip_configuration.0.id
+  ]
+}
+`, rInt, location, rInt, rInt, rInt, rInt, rInt, rInt)
+}
+
+func testAccAzureRMPrivateEndpoint_basic(rInt int, location string) string {
+	standardResources := testAccAzureRMPrivateEndpointTemplate_standardResources(rInt, location)
+
+	return fmt.Sprintf(`
+%s
+
+resource "azurerm_private_endpoint" "test" {
+  name                = "acctestendpoint-%d"
```

Comment:
COuld we include some capitals here?
```suggestion
  name                = "acctest-EndPoint-%d"
```

Reply:
Fixed.